bin/run_epix

```diff
@@ -19,7 +18,7 @@ def pars_args():
     parser.add_argument('--Detector', dest='detector', type=str,
                         action='store', default='XENONnT',
                         help='Detector which should be used. Has to be defined in epix.detectors.')
-    parser.add_argument('--DetectorConfig', dest='detector_config', type=str,
+    parser.add_argument('--EpixConfigOverride', dest='epix_config_override', type=str,
```

Comment:
Don't know if we need to add Epix to the argument, since it is clear that it's a epix parameter or? And I would maybe keep `detector` in to make clear that only the detector config is changed and not the other epix args.

Reply:
No it's not. Because we also have a "fax_config_override". We need to track which one is which else everything becomes a mess.
And "DetectorConfig" is to general, this will be hard to track where it goes